IP-POLICY-GUIDANCE.md

```diff
@@ -0,0 +1,190 @@
+to update them.  If you feel a need to add a _new_ copyright line, see the
+[copyright guidance](#copyrights-and-openjs-foundation-projects).
+1. You can use any of the open source licenses that are pre-approved in the
+[IP Policy](https://ip-policy.openjsf.org) without any additional approvals.
+1. If you need to use a license which isn't in the list, please ask the board.
+1. Your project website needs to have a
+[standard website footer](#copyright-notices-for-website-footers).
+1. If in doubt about anything related to IP, first read the
+[IP Policy](https://ip-policy.openjsf.org), and then
+[ask for help](#getting-help).
+
+There are also some optional things that projects can do which, while not
+mandatory, are referenced in the IP Policy or in this guidance. For example:
+
+1. You may update copyright lines that say "JS Foundation" or
+"Node.js Foundation" to "OpenJS Foundation".  This is a factual update
+reflecting the merged status of the organizations.
+1. You may decide to use the [CLA](https://cla.openjsf.org), or decide to stop
+using one.  This is optional under the OpenJS Foundation IP Policy, and is a
+decision that is to be made by the project's maintainers.
+1. Inserting [SPDX IDs](https://spdx.org/ids-how): While projects decide the
+manner in which they communicate license information, SPDX short identifiers
+are a good option because they improve the accuracy of automated license
+scanners. If you are updating header information anyhow, they are a simple
+one-line addition.
+
+Please reach out to the Foundation with any questions on topics related to the
+IP Policy by emailing
+[legal-questions@lists.openjsf.org](mailto:legal-questions@lists.openjsf.org).
+
+## Copyrights and OpenJS Foundation projects
+
+### Ownership of Copyrights in OpenJS Foundation Contributions
+
+When source code, documentation and other content is contributed to an Open JS Foundation
+project, the copyrights in those contributions remain owned by the original
+copyright holders.
+
+The copyrights are not _assigned_ to OpenJS Foundation. Instead, they are _licensed_ for
+distribution as part of the project. Whether a project uses the <abbr title="Developer Certificate of Origin">[DCO](https://developercertificate.org)</abbr> or the <abbr title="Contributor License Agreement">[OpenJS Foundation CLA](https://cla.openjsf.org)</abbr>,
+the original copyright holders retain their copyrights.
+
+### Copyright Notices
+
+OpenJS Foundation does not require or recommend that every contributor include their
+copyright notice in contributed files. [See below for more details on why
+not.](#why-not-list-every-copyright-holder)
+
+Instead, OpenJS Foundation recommends using a more general statement in a form similar to the
+following (where XYZ is the project's name):
+
+- **Copyright The XYZ Authors.**
+- **Copyright The XYZ Contributors.**
+- **Copyright Contributors to the XYZ project.**
```

Comment:
where should this be put?

Reply:
How about something along the lines of: "If you absolutely must add a copyright line, despite the above guidance that it is not required, please add it in the location specified by the project's open source license."

My rationale in that wording is that the pre-approved licenses all include a generic `Copyright <YEAR> <COPYRIGHT HOLDER>` somewhere in the text.

@ljharb, earlier in the doc, you'd mentioned you want to avoid guidance that results in non-essential boilerplate. Can you please have a look at this too?

Reply:
That works!

Reply:
Cool. 

@tobie, just kidding, I'll take then pen one more time to push this change and THEN it's yours ;-)